src/App.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function filterMoviesList(value: string, moviesList: Movie[]): Movie[] {
+    return moviesList.filter(movie => {
+      return (
+        movie.title.toLocaleLowerCase().includes(value.toLocaleLowerCase())
+        || movie.description
+          .toLocaleLowerCase().includes(value.toLocaleLowerCase())
+      );
+    });
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const prepareQuery = query.trim();
```

Comment:
I think it's better to convert string to lower case here, not in the filterMovieList function.
`const prepareQuery = query.toLowerCase().trim();`

Reply:
agree